File: BasicMaths/ReverseOfNumberN/ReverseOfNumber.ts
class ReverseOfNumber {
  reverseNumber(n: number): void {
    let reversedNumber: number = 0;
    while (n > 0) {
      reversedNumber = reversedNumber * 10 + (n % 10);
      n = Math.floor(n / 10);
    }
    console.log("The reverse of number: " + reversedNumber);
  }
}

const reverse = new ReverseOfNumber();
reverse.reverseNumber(1234567);

File: BasicMaths/ReverseOfNumberN/ReverseOfNumber.py
class ReverseOfNumber:
    def reverseNumber(self,number):
        reversedNumber = 0
        while(number > 0):
            reversedNumber = reversedNumber * 10 + (number % 10)
            number //= 10
        print(reversedNumber)
    
    def run(self):
        n = int(input("Enter the number: "))
        self.reverseNumber(n)

reverse = ReverseOfNumber()
reverse.run()